docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -33,11 +33,117 @@ Understanding GBD's Disease and Health Categories
 GBD Definitions
 ---------------
 
-- Include the GBD definitions for each of these different categories 
+These definitions and others can be found on the `glossary Hub page <https://hub.ihme.washington.edu/pages/viewpage.action?spaceKey=INTRANET&title=IHME+Glossary>`_. 
 
-Understanding the GBD Hierarchy
--------------------------------
+.. list-table:: GBD definitions of terms 
+  :widths: 15 15 15
+  :header-rows: 1
 
-- Showing what is a direct subset, what overlaps, what interacts but is independent 
-- Include some information on modeling implications as a result of this 
+  * - Term 
+    - Definition
+    - Notes
+  * - Cause
+    - A cause of death or disability. Includes injuries, diseases, and conditions. 
+    - See cause list for more details 
+  * - Cause List 
+    - The list of all causes in GBD. It is a mutually exclusive and collectively exhaustive list of hierarchical categories that does not ignore any cause of death. All death and disability within GBD can be found within the cause list. 
+    - 
+  * - Health State 
+    - The current state of health associated with a given cause and sequela.
```

Comment:
Are health states specific to sequela? I think I'm stuck on "cause and sequela" here. Also, is it possible to have multiple health states for a given sequela? For instance -- here is a sequela in the uterine fibroids cause that seems like it has multiple health states (mild abdominal pain AND mild anemia): mild_abdominal_pain_due_to_uterine_fibroids_with_mild_anemia